### ISSUE TITLE
SnackBar is not accessible

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14377]
## Steps to Reproduce

1) Open Gallery App 
2) Linear navigate to Dialog button under Material Components and double tap 
3) Linear navigate to ALERT/ALERT WITH TITLE etc and double tap
4) Tap on ""CANCEL"" or ""DISCARD"" option

User is not notified about the selection on Dialogs page
Actual Result: Selecting ""DISCARD, CANCEL"" option shows a message but it's not conveyed to the user

**Comment by goderbauer** [comment]
1) Open Gallery App 
2) Linear navigate to Snackbar button under Material Components and double tap
3) Double tap on ""SHOW A SNACKBAR"" button 

Message not announced when tapping on ""SHOW A SNACKBAR"" button
Actual Result: Screen reader doesn't announce the message (This is snackbar #1, 2 etc) but it appears at the bottom of the screen
